### PR TITLE
Migrate quickstart guide to DAML assistant

### PR DESCRIPTION
**Pull request by cocreature**
I just went through the quickstart guide on Windows with the new assistant and these are the changes that I had to make to get things working.

We need to figure out how exactly we want to handle the docs migration to the new assistant before merging this.

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/getting-started/quickstart.rst [hunk 1]
```diff
@@ -114,7 +113,7 @@ Run the application using prototyping tools
 
 In this section, you will run the quickstart application and get introduced to the main tools for prototyping DAML:
 
-#. To compile the DAML model, run ``da run damlc -- package daml/Main.daml target/daml/iou``
+#. To compile the DAML model, run ``daml package -o target/daml/iou.dar``
```
Comment (hurryabit) [inline docs/source/getting-started/quickstart.rst]: Wasn't there a PR lately that replaces `target` with `dist`?

Comment (cocreature) [reply docs/source/getting-started/quickstart.rst]: The default if you don’t specify an output path is in `dist` (and will also have a different dar name) but the quickstart example (in particular the codegen) rely on this location so for now I don’t want to change this.

Comment (hurryabit) [reply docs/source/getting-started/quickstart.rst]: Makes sense. Thanks for the explanation.

Comment (associahedron) [inline docs/source/getting-started/quickstart.rst]: Can you change this to `daml build -o target/daml/iou.dar`? The package command is going away / getting hidden in favor of `build`.

Comment (cocreature) [reply docs/source/getting-started/quickstart.rst]: Done

**Comment by leo-da** [comment]
@associahedron can you also rename `quickstart` to `quickstart-java`. I am working on `quickstart-scala` DAML Assistant template, see: #614 + corresponding docs will come soon: #675

**Comment by bethaitman** [comment]
@cocreature - with regards to these docs changes, this all looks sensible, thanks for running through it and getting the guide updated!

@leo-da - I don't think we were planning to provide a different version of this page for Scala? At least that's not what we'd discussed. The point of the quickstart guide is to provide one path that shows the stack off: the user need is to see what you can achieve with DAML, and it doesn't require showing you the different languages you can use to build applications. It's great if we're going to provide an example Scala project, but I'd suggest that it's not integrated as a template - instead, it should be in its own repository, like the [examples using the Java bindings](https://github.com/digital-asset/ex-java-bindings).

**Comment by leo-da** [comment]
@bethaitman my understanding that this `quickstart` folder is also used as a source for the DAML Assistant `quickstart-java` project template. I am adding `quickstart-scala` template in there. So, I thought it would make sense to give both folders very specific names.

**Comment by leo-da** [comment]
@cocreature I am adding `quickstart-scala` folder on the same level with `quickstart`. Is that right or just tell me where to put the new DAML Assistant template.

**Comment by bethaitman** [comment]
@leo-da I'm suggesting _not providing a template at all_ - instead, publishing the example Scala project as its own GitHub repository. (That's what we do with almost all our example projects, with the quickstart as a special exception.) Does that make sense? You can take a look at this repository for an example: https://github.com/digital-asset/ex-java-bindings

**Comment by cocreature** [comment]
Closing in favor of #740

**Pull request closed without merging**